bsp_diff/caas/device/intel/civ/host/vm-manager/0003-SDE-FRP-vm-manager-start_civ.sh-changes.patch

```diff
@@ -13,7 +13,7 @@ Signed-off-by: Xiao He <xiao.he@intel.com>
  2 files changed, 125 insertions(+), 45 deletions(-)
 
 diff --git a/scripts/start_civ.sh b/scripts/start_civ.sh
```

Comment:
Base on "https://github.com/projectceladon/vm_manager/pull/171", do we also need to revert "-cpu host,-waitpkg \" in start_civ.sh file?

Reply:
yes, we have to revert this as well